01-ruby-fundamentals/07-oo-classes.md

```diff
@@ -4,6 +4,7 @@
   - Use our first _instance variables_
```

Comment:
I feel like based on the updates that have been made, we should really update these learning goals to be more specific